drivers/src/qemuexit.rs

```diff
@@ -0,0 +1,35 @@
+use qemu_exit;
+use qemu_exit::QEMUExit;
+
+use crate::Driver;
+
+pub struct QemuExit {
+    address: usize,
+    #[cfg(target_arch = "riscv64")]
+    inner: qemu_exit::RISCV64,
+}
+
+impl QemuExit {
+    #[allow(clippy::new_without_default)]
```

Comment:
Urgh :DD Why not have a `Default` implementation? it makes sense for other types who'd want to use it, and it allows cleaner syntax overall in the project

And disabling clippy warnings is a capital crime

Reply:
Fixed